packages/daimo-contract/package.json

```diff
@@ -5,15 +5,17 @@
   "types": "./dist/index.d.ts",
   "main": "./dist/index.js",
   "scripts": {
-    "codegen": "wagmi -v && wagmi generate",
+    "codegen": "wagmi -v && wagmi generate && tsx src/shovel/index.ts",
     "build": "tsc"
   },
   "author": "",
   "license": "GPL-3.0-or-later",
   "devDependencies": {
+    "@indexsupply/shovel-config": "^0.0.9",
     "@pimlico/erc20-paymaster": "^0.1.1",
     "@tsconfig/node20": "^20.1.0",
     "@wagmi/cli": "^1.4.1",
+    "tsx": "^4.7.2",
```

Comment:
we're using ts-node for this in other spots, is tsx better?

Reply:
`ts-node` has some outstanding bugs in interop with packages built with `bun`, `tsx` seems to be the recommended way to fix them.